content/docs/apps/govcloud.md

```diff
@@ -85,7 +85,7 @@ Any labor involved from your project team is the responsibility of your project.
 
 1. Fill out the [organization request form](https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLSd4HmcGfJW3EBnpewTFDD-urRFPp1LN0DcwNB_FxZgUn8ho9g/viewform?c=0&w=1).
     * Specify that the org should be created in `GovCloud`.
-    * An admin will confirm the information, and [create the org]({{< relref "docs/ops/create-org.md#non-sandboxes" >}}) for you.
+    * An admin will confirm the information, [create the org]({{< relref "docs/ops/create-org.md#non-sandboxes" >}}) for you, and notify you. This should take less than one business day.
```

Comment:
In the interest of managing expectations, how about `This should happen within one business day`? That would sound more like time <= 1 business day instead of time < 1 business day.